salt/modules/zcbuildout.py

```diff
@@ -299,7 +287,7 @@ def _Popen(
     directory = os.path.abspath(directory)
     if isinstance(command, list):
         command = " ".join(command)
-    LOG.debug("Running {0}".format(command))  # pylint: disable=str-format-in-logging
+    LOG.debug("Running {}".format(command))  # pylint: disable=str-format-in-logging
```

Comment:
```suggestion
    LOG.debug("Running %s", command)
```

Reply:
We can't do that here, this module is using some weird custom logger. Looks like it's used to get logging in the execution function returns. I'm not going to untangle this mess in the PR.